src/Epicea/EpRBPropagateRefactoring.class.st

```diff
@@ -40,6 +45,11 @@ EpRBPropagateRefactoring >> setOption: aString toUse: aBlockClosure [
 	targetRefactoring setOption: aString toUse: aBlockClosure 
 ]
 
+{ #category : #accessing }
+EpRBPropagateRefactoring >> transform [
```

Comment:
Hi evelyn 
Are you sure that we need this method?

Reply:
I already corrected it, it was an error that I put while testing code critics for Ale Cossio's project